### DISCUSSION TITLE
SMTP Usage

### DISCUSSION THREAD

**Discussion by Elio-FairlyMade** [drakkan/sftpgo#1549] [Q&A] [answered]
Hello,

I've successfully configured my SFTPGo server on docker and now I'm wondering the usages of the SMTP configuration.
Is it only used by the Event actions ?
I was wondering if it was possible to send a mail once a user is created.

Thanks a lot :)

**Comment by drakkan** [comment, marked as answer]
Hello, yes it is possible using the EventManager

**Reply by Elio-FairlyMade** [reply]
Thanks
